.github/workflows/codestyle.yml

```diff
@@ -29,3 +33,74 @@ jobs:
 
       - name: Run Black
         run: black --check tardis
+  ruff: 
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.12.3' 
+      
+      - name: Install Ruff
+        run: pip install ruff==0.4.9
```

Comment:
update it.

Reply:
done in [`fea7705` (#2660)](https://github.com/tardis-sn/tardis/pull/2660/commits/fea77053fd38874079a558034e15c96240638210)